go/libraries/doltcore/env/environment.go

```diff
@@ -155,6 +160,11 @@ func Load(ctx context.Context, hdp HomeDirProvider, fs filesys.Filesys, urlStr,
 	return dEnv
 }
 
+func GetDefaultInitBranch(dEnv *DoltEnv) string {
```

Comment:
when used in other packages usage looks like:

`env.GetDefaultInitBranch(dEnv)`

If it were a member of dEnv it would be:

`dEnv.GetDefaultInitBranch()`

which feels a bit better.

Reply:
I changed the param to be a `config.Config`